types/predict.ts

```diff
@@ -0,0 +1,48 @@
+export type PredictServiceBasicValueType =
```

Comment:
I noticed none of the new types that are added seem to be exported from `types/index.ts` which means the user won't have access to them. I assume you want the developer to have access to some of them, can you make sure they're exported from `types/index.ts`?